### PR TITLE
K8s-logging: Demo Logger object bound to PVC

### PR DESCRIPTION
**Pull request by gargpalak048**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ Logging Service to demonstrate the distributed logging with log collector.
       git clone https://github.com/sumedhak27/Containerized-Distributed-Logger.git
       cd Containerized-Distributed-Logger
       ```
+    - Build docker image of logger application
+      ```bash
+      docker build -f Dockerfile -t demo-logger:1.1 .
```
Comment (sumedhak27) [inline README.md:28]: we should keep the <version_num> instead of 1.1

Comment (sumedhak27) [inline README.md]: line number 51 https://github.com/sumedhak27/Containerized-Distributed-Logger/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R51
needs a change 
`less /var/log/demoLogger/[container_id].log`

File: logger.py [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
+def infi_logger(logger=None):
     i = 0
     while True:
-        ctime = datetime.datetime.now().strftime("%b %d %Y %H:%M:%S")
-        logline = f"{ctime}: [{hostname}] Logging count {i}"
-        if location == "terminal":
-            print(logline)
+        logline = f"Logging count {i}"
+        if logger:
+            logger.debug('This message should go to the log file')
```
Comment (sumedhak27) [inline logger.py]: why logging this to the file??

File: README.md (hunk 1 above)
Comment (sumedhak27) [inline README.md]:
```suggestion
      # version_num/tag of the image and mentioned in the deployment.yaml should be same.
      docker build -f Dockerfile -t demo-logger:[version_num/tag] .
```

Comment (gargpalak048) [reply README.md]: done

File: logger.py (hunk 2 above)
Comment (gargpalak048) [reply logger.py]: For debug purpose.
Removed now

**Review by sumedhak27 — APPROVED** [review]

**Pull request merged**